config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -52,6 +52,7 @@ presubmits:
 
 
   - name: pull-kubernetes-e2e-kind-canary
+    cluster: k8s-infra-prow-build
```

Comment:
Make the canary run in the same place w/ same limits as pull-kubernetes-e2e-kind